### ISSUE TITLE
BT-706 (Winner Parent Country); Description; Sometimes impossible to tell

### ISSUE THREAD

**Issue by rantati7** [eForms/eForms#13]
In case of publicly traded companies this information is next to impossible to get and also impossible for a CA to investigate whether the information is correct. 

This would be "intersting" information but if someone wants to investigate this could it be done using the organisation indentifier? I think this is too much administrative burden to CAs and should be abolished from the eForms.

**Comment by rantati7** [comment]
If this question is necessary it should be included in ESPD

**Comment by RolfKewitz** [comment]
Should the winner parent country only be known by the contracting authority ESPD is sufficent. If it has be be published, too, it should be known under which conditions this (conditional mandatory) field shall be mandatory.

**Comment by JachymHercher** [comment]
This field was requested to better understand (and better negotiate) EU's trade agreements. Since this requires having complete and comparable data, it needs to be in the TED forms, not just the ESPD (because the ESPD is not always published.)

From the accuracy point of view though, you are totally right that the ESPD would be a good source of this data. (FYI: @ec-mcs) However, using the ESPD means we request this information from everyone, while it is needed only from the contractor/winner. Doesn't that create unnecessary administrative burden? (As with many other fields of the ESPD, this information should eventually come from other administrative databases, in this case the ultimate benefitiary ownership register established under the anti-money laundering directive.) 

The BT is CM just to ensure that the field only applies when the organisation is a contractor (and not a buyer, for example). In other words, the CM stands for "mandatory if the organisation is a contractor".

**Comment by ognyami** [comment]
In BT-706 (“Winner Parent Company”), the description must be reformulated or the BT should be removed. If kept with the current definition of "parent company" it will provide distorted and easy to manipulate information. 
Example 1: If Company A domiciled in Country A is owned by ultimate parent Company B domiciled in Country B – the “Winner Parent Country” will be Country B; but if the ownership of the same Company A is split equally (50-50) between Company B1 and Company B2 (i.e. none of the companies owns more than 50%), both domiciled in Country B, the “Winner Parent Country” will be either Country A  or the field should be left blank, if this is possible at all (and depending on the possible interpretations of the text).
Example 2: if Company A domiciled in Country A is owned by ultimate parent Company B domiciled in Country B (with a 100% ownership of a natural person NP residing in Country B) – the “Winner Parent Country” will be Country B; but if in the same example Company B is domiciled in Country A, instead in Country B (with a 100% ownership of the same natural person NP residing again in Country B), the “Winner Parent Country” will be Country A. Thus, simple choice by the beneficial owner how to structure his ownership will provide an artificially different result in the two cases.
It is also not clear how the contracting authority would know who the “winner parent company” is – it seems the only way to get this information would be self-declaration from the winner, which could not always be reliable (they simply might not know). If the BT is to be kept, it would be better to refer to a term already established by the EU legislation – for example _"The country/-ies where **the beneficial owner(s)** of the winner are residents, and if the winner or (some of) its beneficial owner(s) is/are company/-ies listed on a regulated market – the country/ies of registration of the listed company/-ies",_ using the definition for beneficial owners of DIRECTIVE (EU) 2015/849 on the prevention of the use of the financial system for the purposes of money laundering or terrorist financing. Under the recent update of the Directive through the 5th Anti Money Laundering Directive (Directive (EU) 2018/843 - OJ L 156, 19.6.2018, p. 43–74) the registers containing information of the beneficial owners of the companies registered in the EU shall become public in 2020, so for the majority of the winners the information who their beneficial owners are will be precise, identifiable and easily verifiable by the time the Implementing Regulation enters into force.

**Comment by JachymHercher** [comment]
You are right that getting this information from registers established under the anti-money laundering directive is the best approach - it will lead to lowest administrative burden as well as, perhaps, highest quality of data. These registers should in any case be used by buyers to verify provisions on conflict of interest and exclusion criteria. Consequently, we propose the following definition: 

"The nationality (or nationalities) of the winner's (or winners') beneficiary owner (or beneficiary owners), as listed in the registers established by anti-money laundering legislation."

In line with the definition, we have changed the BT to repeatable.

(For reference: in the [consolidated version of the anti-money laundering directive](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:02015L0849-20180709): 

- Beneficiary owners are defined in Art. 3(6) 
(note: most importantly, ownership is taken into account only if it is above 25%). 
- The registers are established in Art. 30(3).
- Information to be made publicly available is established in Art. 30(5) 
(note: most importantly, this includes nationality and country of residence)).
